### PR TITLE
docs: don't apply layouts to html formatter example

### PR DESCRIPTION
**Pull request by Tanujkanti4441**
#### Prerequisites checklist

- [x] I have read the [contributing guidelines](https://github.com/eslint/eslint/blob/HEAD/CONTRIBUTING.md).

#### What is the purpose of this pull request? (put an "X" next to an item)

[x] Documentation update
[ ] Bug fix ([template](https://raw.githubusercontent.com/eslint/eslint/HEAD/templates/bug-report.md))
[ ] New rule ([template](https://raw.githubusercontent.com/eslint/eslint/HEAD/templates/rule-proposal.md))
[ ] Changes an existing rule ([template](https://raw.githubusercontent.com/eslint/eslint/HEAD/templates/rule-change-proposal.md))
[ ] Add autofix to a rule
[ ] Add a CLI option
[ ] Add something to the core
[ ] Other, please explain:

#### What changes did you make? (Give an overview)
The table is going out of the screen in mobile size.

IMG_20221126_132158

The font weight of header and side nav is thinner than the usual one and in dark mode some text are not properly visible

Screenshot 2022-11-25 233249

So, as a change i removed some CSS properties like color from the body of this page and for table i decreased the padding of row and font size of table header for particular screen size.

#### Is there anything you'd like reviewers to focus on?
https://eslint.org/docs/latest/user-guide/formatters/
https://eslint.org/docs/latest/user-guide/formatters/html-formatter-example

### CODE REVIEW

**Review by mdjermanovic — CHANGES_REQUESTED** [review]
This file is autogenerated output of `eslint -f html`. It is not meant to be a separate documentation page on the eslint.org website, but only to be included in an iframe on the [formatters page](https://eslint.org/docs/latest/user-guide/formatters/#html).

The problem is actually that the file is processed by eleventy so on the site it has headers, menus etc. while I think it should be just copied as-is so that it looks like on the [old formatters page](https://archive.eslint.org/docs/user-guide/formatters/#html).

**Comment by Tanujkanti4441** [comment]
@mdjermanovic thanks for review.
My description and screenshots of PR may sounding like it is focused on the [Formatter-example](https://eslint.org/docs/latest/user-guide/formatters/html-formatter-example) page but this PR was mainly made to fix the UI related issues of that iframe section of [Formatters](https://eslint.org/docs/latest/user-guide/formatters) page.

As I am getting your review, you want me to remove rest of the html elements except the table from the formatter's page iframe section as in the old one have

**Comment by mdjermanovic** [comment]
Yes, the file should remain unchanged in the build output. I think this was accidentally changed by https://github.com/eslint/eslint/pull/16484.

A quick fix could be to add `docs/src/user-guide/formatters/html-formatter-example.json` file with the following content:

```json
{
    "layout": false
}

```

File: docs/src/user-guide/formatters/html-formatter-example.html [hunk 1]
```diff
@@ -5,18 +5,18 @@
...
+        <link rel="stylesheet" type="text/css" href="{{ '/assets/css/tokens/themes.css' | url }}">
```
[the remark this answers is not in the document]
Comment (Tanujkanti4441) [reply docs/src/user-guide/formatters/html-formatter-example.html]: Done! Have removed everything from html-formatter-example.html file that I added

**Review by mdjermanovic — APPROVED** [review]
LGTM, thanks!

**Pull request merged**